scenarios/kubernetes_e2e.py

```diff
@@ -230,6 +230,7 @@ def add_environment(self, *envs):
         Host-specific environment variables are ignored."""
         # TODO(krzyzacy) change this to a whitelist?
         docker_env_ignore = [
+            'E2E_PUBLISH_PATH',  # Use --publish flag
```

Comment:
agreed. this list is intended for things which shouldn't get passed into the docker container because they don't make sense (or there are already default values that shouldn't get overridden), not deprecated config variables.